### ISSUE TITLE
Not compiling in Xcode 8

### ISSUE THREAD

**Issue by protspace** [ReSwift/ReSwift#153]
```
.../Action.swift:56:11: Expected 'let' in conditional
.../Store.swift:147:39: Function types cannot have argument label 'state'; use '_' instead
.../Store.swift:147:53: Function types cannot have argument label 'store'; use '_' instead
.../Store.swift:150:9: Function types cannot have argument label 'state'; use '_' instead
.../Store.swift:151:9: Function types cannot have argument label 'store'; use '_' instead
.../Store.swift:152:9: Function types cannot have argument label 'actionCreatorCallback'; use '_' instead
.../StoreType.swift:162:37: Function types cannot have argument label 'state'; use '_' instead
.../StoreType.swift:162:51: Function types cannot have argument label 'store'; use '_' instead
.../StoreType.swift:169:42: Function types cannot have argument label 'state'; use '_' instead
.../StoreType.swift:169:56: Function types cannot have argument label 'store'; use '_' instead
.../StoreType.swift:170:5: Function types cannot have argument label 'actionCreatorCallback'; use '_' instead
/Users/eugensagidulin/repos/Weezsport/Pods/ReSwift/ReSwift/Utils/TypeHelper.swift:24:30: Function types cannot have argument label 'action'; use '_' instead
/Users/eugensagidulin/repos/Weezsport/Pods/ReSwift/ReSwift/Utils/TypeHelper.swift:24:46: Function types cannot have argument label 'state'; use '_' instead
.../CombinedReducer.swift:34:34: Missing argument label 'action:' in call
.../Middleware.swift:13:64: Single argument function types require parentheses
.../Action.swift:63:39: Cannot convert value of type '[String : AnyObject]?' to expected argument type 'AnyObject?'
.../Action.swift:65:26: Value of type 'String' does not conform to expected dictionary value type 'AnyObject'
```

**Comment by devladinci** [comment]
@protspace `pod 'ReSwift', :git => 'https://github.com/ReSwift/ReSwift.git'` will get you back to work :) I guess they will push new pod ver soon

**Comment by Ben-G** [comment]
@protspace @devladinci Need to create a new release with support for Xcode 8. `master` branch already has it. 

Hopefully should be able to get this done tonight.

**Comment by Ben-G** [comment]
This should be fixed with #154. Please let me know if you're still running into issues!
